service/dap/server.go

```diff
@@ -858,10 +890,7 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 	}
 
 	s.config.ProcessArgs = append([]string{program}, args.Args...)
-	s.config.Debugger.WorkingDir = filepath.Dir(program)
-	if args.Cwd != "" {
-		s.config.Debugger.WorkingDir = args.Cwd
-	}
+	s.config.Debugger.WorkingDir = args.Cwd // cwd: "" == "."
```

Comment:
Why not make "." explicit instead of relying on the comment?

Reply:
Done.